coding-concepts.js

```diff
@@ -12,43 +12,59 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // a) Your answer:
 // b) Verify and explain:
 
+
+
+
+
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I beleive this while log 10 because we are asking for a .length. If we asked for the total index we would get 9 because counting starts at 0 for indexes.
+// b) Verify and explain: I was correct. If an array has 10 variables, the .length is 10, but because arrays are zero indexed the first value position starts at 0 - 9 in this problem.
+
+
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I think that this will return the index of "o". If the greeting index[4] starts counting at zero, then the 5th variable in the string will be returned.
+// b) Verify and explain: "o" was correct. Whenever our code is running an index value at a specific position, the first position in the variable is 0. ex.) H=0, e=1, l=2, l=3, and 0=4.
```

Comment:
Nicely done! I think I know what you were trying to say with this one, just be careful using variable since it's a specific thing in JS. You could instead say the 5th character ...